### PR TITLE
Separate examples directory 

### PR DESCRIPTION
**Pull request by dotansimha**
- Separate examples to another directory
- Simplify installation and execution of examples (using `npm install` instead of `npm link`) 
- Updated README

### CODE REVIEW

File: examples/editor-example/package.json [hunk 1]
```diff
@@ -24,17 +24,17 @@
...
+    "wix-rich-content-editor": "file:../../packages/editor",
+    "wix-rich-content-plugin-divider": "file:../../packages/plugin-divider",
+    "wix-rich-content-plugin-emoji": "file:../../packages/plugin-emoji",
+    "wix-rich-content-plugin-gallery": "file:../../packages/plugin-gallery",
+    "wix-rich-content-plugin-hashtag": "file:../../packages/plugin-hashtag",
+    "wix-rich-content-plugin-html": "file:../../packages/plugin-html",
+    "wix-rich-content-plugin-image": "file:../../packages/plugin-image",
+    "wix-rich-content-plugin-link": "file:../../packages/plugin-link",
+    "wix-rich-content-plugin-mentions": "file:../../packages/plugin-mentions",
+    "wix-rich-content-plugin-video": "file:../../packages/plugin-video"
```
Comment (bchmn) [inline examples/editor-example/package.json]: what happens when we want to deploy this example to production for a demo?

Comment (dotansimha) [reply examples/editor-example/package.json]: It should work as long we deploy it from the entire project repository.

Comment (bchmn) [reply examples/editor-example/package.json]: I'm pretty sure our CI doesn't know how to do that

**Comment by bchmn** [comment]
since there is a dedicated `examples` directory I think we can omit the `-example` from the names:
`editor-example` -> `editor`
`viewer-example` -> `viewer`
`editor-tpa` can remain the same.
WDYT?

File: examples/editor-example/package.json (hunk 1 above)
Comment (dotansimha) [reply examples/editor-example/package.json]: What do you mean?
The CI works per-repository, so it should still use the entire repo in order to publish the examples

Comment (bchmn) [reply examples/editor-example/package.json]: fair enough, worst case scenario we'll make a couple of adjustments for CI further down the road

**Comment by bchmn** [comment]
cool, I'm waiting for the directory renames and then Ill merge

**Comment by dotansimha** [comment]
@bchmn done :)

**Pull request merged**